docs/options.md

```diff
@@ -113,6 +117,9 @@ CIBW_SKIP: cp36-manylinux*
 # Only build on Python 3 and skip 32-bit builds
 CIBW_BUILD: cp3?-*
 CIBW_SKIP: "*-win32 *-manylinux_i686"
+
+# Only build PyPy and CPython 3
+CIBW_BUILD: pp* cp3?-*
```

Comment:
Maybe also worth adding a skip PyPy example e.g.

```yaml
# Disable building PyPy wheels on all platforms
CIBW_SKIP: pp*
```

Reply:
OK, will add this tomorrow.

Reply:
Not sure where to add it, but it's now just a the bottom of the list.